test/fixtures/webpack-message-formatting/__snapshots__/index.test.js.snap

```diff
@@ -58,31 +58,38 @@ Syntax error: Unexpected } (3:2)
 
 exports[`formats eslint error 1`] = `
 Object {
-  "stderr": "Creating an optimized production build...
-Failed to compile.
+  "stderr": "",
+  "stdout": "Creating an optimized production build...
+Compiled successfully.
 
-./src/App.js
-  Line 4:13:  'b' is not defined  no-undef
```

Comment:
Why did this go away?

Reply:
Again a result of `../node_modules/.bin/jest -w 2` with `-u` flag I ran.

Reply:
For some reason, ESLint is reporting no-undef and no-unused-vars.

Reply:
Like I wrote above, snapshots shouldn't just be "updated", we need to understand what changed and why. Unless you're saying they already fail on master.